holoviews/plotting/mpl/plot.py

```diff
@@ -916,7 +916,7 @@ def _compute_gridspec(self, layout):
 
         # Create title handle
         if self.show_title and len(self.coords) > 1:
-            title = self.handles['fig'].suptitle('', **self._fontsize('title'))
+            title = self.handles['fig'].suptitle('', y=1.05, **self._fontsize('title'))
```

Comment:
Yea, generally avoids overlap between the plot title and the overall layout title. Not sure how best to provide control over it.